light/rpc/client.go

```diff
@@ -591,6 +592,18 @@ func (c *Client) ProveShares(
 	return res, err
 }
 
+// ProveSharesV2 returns a proof of inclusion for a share range to the data root.
+// Note: this proof is composed of multiple proofs.
+func (c *Client) ProveSharesV2(
+	ctx context.Context,
+	height uint64,
+	startShare uint64,
+	endShare uint64,
```

Comment:
Mind adding some comments about the values permitted for the start and end shares?